cheroot/ssl/pyopenssl.py

```diff
@@ -150,14 +150,26 @@ def readline(self, size=-1):
             size,
         )
 
-    def sendall(self, *args, **kwargs):
-        """Send whole message to the socket."""
+    def read(self, *args, **kwargs):
+        """Read from the wrapped socket, with retry."""
         return self._safe_call(
-            False,
-            super(SSLFileobjectMixin, self).sendall,
+            True,
+            super(SSLFileobjectMixin, self).read,
             *args, **kwargs
         )
 
+    def sendall(self, *args, **kwargs):
+        """Send whole message to the socket.
+
+        Not supported due to https://github.com/pyca/pyopenssl/issues/176.
+        Until that bug is fixed, sendall() may throw SSL.WantWriteError, but
+        there is no correct way to retry the call because we don't know how
+        many bytes were already transmitted. We could work around this by
+        reimplementing sendall() using send(), but we don't actually use
+        sendall() anywhere.
+        """
```

Comment:
Could you please make this a code comment, I don't think it belongs in a docstring.
```suggestion
        """
        # Until that bug is fixed, sendall() may throw SSL.WantWriteError, but
        # there is no correct way to retry the call because we don't know how
        # many bytes were already transmitted. We could work around this by
        # reimplementing sendall() using send(), but we don't actually use
        # sendall() anywhere.
```

Reply:
P.S. FYI it's a good tone to accept suggestions via GH UI because that preserves authorship.